packages/core/data/oot/world/overworld.yml

```diff
@@ -1429,6 +1429,11 @@
   locations:
     "Zora Domain Waterfall Chest": "is_child"
     "Zora Domain Diving Game": "is_child && can_use_wallet(1) && soul_npc(SOUL_NPC_ZORA)"
+    "Zora Domain Diving Game Green Rupee": "is_child && can_use_wallet(1) && soul_npc(SOUL_NPC_ZORA)"
+    "Zora Domain Diving Game Blue Rupee": "is_child && can_use_wallet(1) && soul_npc(SOUL_NPC_ZORA)"
+    "Zora Domain Diving Game Red Rupee": "is_child && can_use_wallet(1) && soul_npc(SOUL_NPC_ZORA)"
+    "Zora Domain Diving Game Purple Rupee": "is_child && can_use_wallet(1) && soul_npc(SOUL_NPC_ZORA)"
+    "Zora Domain Diving Game Huge Rupee": "is_child && can_use_wallet(1) && soul_npc(SOUL_NPC_ZORA)"
```

Comment:
This would be better extracted to an area dedicated to the diving game (but not urgent)

Reply:
Noted for next time. Would the same ideally be done for the fishing pond fish and rewards?

Reply:
Every time it avoids excessive repeats essentially. But yeah, not urgent, we still have a lot of these repeats.